src/main/java/reposense/report/CommitReportJson.java

```diff
@@ -18,6 +20,27 @@
     private final Map<Author, Float> authorContributionVariance;
     private final Map<Author, String> authorDisplayNameMap;
 
+    /**
+     * Empty constructor to construct an empty commit report.
+     */
+    public CommitReportJson() {
```

Comment:
seems okay for a temporary solution, but this adds on to the maintenance work.
I think it would probably be better if we handle this on the ui side.

Reply:
This solution is to handle the UI(dashboard) side, where we show in the dashboard that we failed to download this repo.
Also, currently there is logging when `GitDownloader` fails, perhaps we can add on to that logged error message that we are creating an empty report folder for them?

Reply:
> *Edit: oh, perhaps you meant if we failed to find the folder in the frontend side, then handle it over there? This would also handle the IOException if we fail to create the report files for whatever reason.

Yes, we can create and attach a feedback field on the RepoConfig, so that when it fails to load on the frontend, we can show that feedback on the frontend by default. 

It would be nice to not treat the 2 exception too differently.

For now, shall we just remove the repoconfig from the list (so that it won't be generated in the summary) since user would've gotten the error message from terminal?